charter.html

```diff
@@ -237,11 +255,11 @@ <h3 id="out-of-scope">
             algorithms.
           </p>
           <p>
-            To avoid overlap with existing work, generic primitives used by
-            traditional machine learning algorithms such as base linear algebra
-            operations are out of scope. The WebGL and WebGPU shaders and
-            WebAssembly SIMD are expected to address these requirements, see
-            the Coordination section for details.
+            To avoid overlap with existing work, alignment with the Basic
+            Linear Algebra Subprograms (BLAS) interface is out of scope. The
+            WebGL and WebGPU shaders and WebAssembly SIMD are expected to
```

Comment:
Since we already removed WebGL interop from the spec, should we also remove it here from the charter? Preferably noted as out of scope.

Reply:
Thanks for the suggestions. I pushed https://github.com/w3c/machine-learning-charter/pull/19/commits/33e2eb59d47a8fcc6d722dbd9c1475c663a2900c that should address your comments. Namely:

I removed the WebGL reference from this BLAS compatibility context and added the following text following this paragraph:

>Interoperability between the WebNN and WebGL APIs is out of scope.

With this change, the only other reference to WebGL in addition to the above out of scope statement remains in the Motivation and Background section as follows:

>Currently, machine learning inference in the browser uses the WebGL graphics API with limited or no access to platform capabilities beneficial for ML such as CPU parallelism, general-purpose GPU, or dedicated ML hardware accelerators.

I think this is appropriate for context setting and to give a historical perspective. With WebNN API becoming more broadly available, the need to use WebGL API for ML workloads is expected to diminish over time.